src/include/ipc4/copier.h

```diff
@@ -119,6 +119,14 @@ enum ipc4_copier_features {
 	IPC4_COPIER_FAST_MODE = 0
 };
 
+#if CONFIG_HOST_DMA_STREAM_SYNCHRONIZATION
+#define HDA_SYNC_FPI_UPDATE_GROUP 0
+struct ipc4_copier_sync_group {
+	uint32_t group_id;
+	uint32_t fpi_update_period_usec;
+} __packed __aligned(4);
```

Comment:
for unint32 started structure, compiler should always allocated a 4 aligned address, right?
I mean __aligned(4) is redundant.

Reply:
@btian1 its best to be explicit as we will be compiling on various 32/64 bit compilers.